### DISCUSSION TITLE
TS2 Compound Remake

### DISCUSSION THREAD

**Discussion by masterpl4n** [nzp-team/nzportable#864] [Map Releases]
### [DOWNLOAD HERE](https://github.com/user-attachments/files/16534543/ts2zombies.zip) ###

![fte-20240807155409-0](https://github.com/user-attachments/assets/f6ce816a-f6bb-4b33-aad6-cd06a4c397a9)

Based on the classic challenge from TimeSplitters 2 shown [here](https://youtu.be/isBb8jlLqcs?si=Uaeb6vRQHF5wXgwx). Not a direct port, had to remove the stairs for game flow reasons, but still accurate enough. There's a secret room with PAP and Speed Cola, can you find it?

Tested on PC and PSP 1000.

![fte-20240807155428-0](https://github.com/user-attachments/assets/726c5f38-e5d8-4ee9-9f5a-b872d9941a8f)

![fte-20240807155603-0](https://github.com/user-attachments/assets/83977f6c-aca8-444e-ac10-294291dad6f2)

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org
